check_license.sh

```diff
@@ -102,7 +102,7 @@ for file in $(find . -type f -iname 'LICEN[SC]E' -o -iname 'LICEN[SC]E.*' -o -in
         echo >&2 "$file has missing or wrong entry in $CHKSUM_FILE"
         ret=1
     fi
-done
+done < <(find . \( -type f -iname 'LICEN[SC]E' -o -iname 'LICEN[SC]E.*' -o -iname 'COPYING' \) -and -not -iname 'LICEN[SC]E.go' -and -not -iname 'LICEN[SC]E.c' -and -not -iname 'LICEN[SC]E.cpp')
```

Comment:
Why put it down here, and use all this shell magic? It's harder to read...

Reply:
iterating over find results is a common mistake.
for example:
```bash
#mkdir /tmp/d; for i in {1..4}; do touch /tmp/d/"ok $i"; done; for i in `find /tmp/d`; do echo "\"$i\""; done
"/tmp/d"
"/tmp/d/ok"
"1"
"/tmp/d/ok"
"2"
"/tmp/d/ok"
"3"
"/tmp/d/ok"
"4"
```

Reply:
Ah, the `read` makes it line based. I didn't think of that. Clever!

Reply:
thank you for the review!